core/metrics-core-api/src/main/java/org/hawkular/metrics/core/api/MetricsService.java

```diff
@@ -84,6 +84,15 @@
 
     Observable<Metric> findMetrics(String tenantId, MetricType type);
 
+    /**
+     *
+     * @param tenantId
+     * @param tags
+     * @param type Optional MetricType, if null is given all matching metrics are returned
```

Comment:
Le 08/07/2015 11:55, Michael Burman a écrit :
> No, as the tags can further filter the results.

Understood, I had a closer look at MetricServiceImpl:
https://github.com/hawkular/hawkular-metrics/pull/280/files#diff-04e1ad2ad2cadc473893730a61b3e271R475

But then why return user types only when the tags query param is null? 
Or maybe I missed something?